plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/IdentityCacheMapping.java

```diff
@@ -24,12 +24,12 @@
      * metadata then we should have {@link IdentityCacheKey} instance so
      * we could cache proper metadata for given {@link ConnectorSession}.
      */
-    abstract class IdentityCacheKey
```

Comment:
The point of abstract class here was to force implementation of hashCode/equals.

Reply:
That's what I though too, actually